api/models.py

```diff
@@ -133,7 +139,10 @@ class Node(DatabaseModel):
     )
     status: Optional[StatusValues] = Field(
         default=StatusValues.PENDING,
-        description='Status of test result'
+        description='Status of test'
```

Comment:
It's actually not a test any more, but status of the node. This is a detail, we may not need to change it now but it'll be interesting to see how things evolve when we have actual test nodes and other kinds of nodes.

Reply:
Updated the description for `status` and `result`.